resources/windows/wix/customactions.wxs

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Include_CustomActions" />
+
+    <CustomAction Id="SaveCmdLineInstallDir"
+      Property="CMDLINE_INSTALLDIR"
+      Value="[INSTALLDIR]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLineInstallDir"
+      Property="INSTALLDIR"
+      Value="[CMDLINE_INSTALLDIR]"
+      Execute="firstSequence" />
```

Comment:
So, from what I understand, every project will need this file with at least these lines (6-13) in it. Is that correct?